### PR TITLE
Add timeout to log

### PR DESCRIPTION
**Pull request by RamLavi**
This PR is printing the timeout to as part of the input param logging.

### CODE REVIEW

**Comment by RamLavi** [comment]
Passed on CNV 4.14 (nightly 4.14.0-0.nightly-2023-07-20-215234) with [OCPBUGS-15102](https://issues.redhat.com/browse/OCPBUGS-15102)'s workaround:
```
docker run --rm \
           --volume /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:Z \
           --volume /home/ralavi/.kube:/root/.kube:Z,ro \
           --workdir /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup \
           -e KUBECONFIG=/root/.kube/kubeconfig \
           -e TEST_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup:latest \
           -e TEST_NAMESPACE=dpdk-checkup-ns-1 \
           -e NETWORK_ATTACHMENT_DEFINITION_NAME=dpdk-sriovnetwork-ns-1 \
           -e TRAFFIC_GEN_IMAGE_URL=quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main \
           -e VM_CONTAINER_DISK_IMAGE_URL=quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main \
           docker.io/library/golang:1.20.6 go test ./tests/... -test.v -test.timeout=1h -ginkgo.v -ginkgo.timeout=1h
=== RUN   TestKubevirtDpdkCheckup
Running Suite: KubevirtDpdkCheckup Suite - /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests
==============================================================================================================
Random Seed: 1690797780

Will run 1 of 1 specs
------------------------------
[BeforeSuite] 
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/test_suite_test.go:60
[BeforeSuite] PASSED [0.002 seconds]
------------------------------
Execute the checkup Job should complete successfully
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/checkup_test.go:79
• [323.736 seconds]
------------------------------

Ran 1 of 1 Specs in 323.738 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestKubevirtDpdkCheckup (323.74s)
PASS
ok  	github.com/kiagnose/kubevirt-dpdk-checkup/tests	323.763s

```

**Comment by RamLavi** [comment]
```
2023/07/31 11:48:58 kubevirt-dpdk-checkup starting...
2023/07/31 11:48:58 Using the following config:
2023/07/31 11:48:58 "timeout": "10m0s"
2023/07/31 11:48:58 "networkAttachmentDefinitionName": "dpdk-sriovnetwork-ns-1"
2023/07/31 11:48:58 "trafficGenContainerDiskImage": "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main"
2023/07/31 11:48:58 "trafficGenTargetNodeName": ""
2023/07/31 11:48:58 "trafficGenPacketsPerSecond": "8m"
2023/07/31 11:48:58 "TrafficGenEastMacAddress": "50:2c:81:26:e9:01"
2023/07/31 11:48:58 "TrafficGenWestMacAddress": "50:cb:f3:58:e2:02"
2023/07/31 11:48:58 "vmUnderTestContainerDiskImage": "quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main"
2023/07/31 11:48:58 "vmUnderTestTargetNodeName": ""
2023/07/31 11:48:58 "VMUnderTestEastMacAddress": "60:46:6f:89:02:01"
2023/07/31 11:48:58 "VMUnderTestWestMacAddress": "60:0d:66:85:32:02"
2023/07/31 11:48:58 "testDuration": "1m0s"
2023/07/31 11:48:58 "portBandwidthGbps": "10"
2023/07/31 11:48:58 "verbose": true
```

File: pkg/mainflow.go [hunk 1]
```diff
@@ -66,6 +66,7 @@ func Run(rawEnv map[string]string, namespace string) error {
 
 func printConfig(checkupConfig config.Config) {
 	log.Println("Using the following config:")
```
Comment (orelmisan) [inline pkg/mainflow.go:68]: Please consider taking the base config as a parameter, so if more fields will be added to it in the future - we could print them as well:
```suggestion
func printConfig(baseConfig kconfig, checkupConfig config.Config) {
	log.Println("Using the following config:")
	log.Printf("Timeout: %q", baseConfig.Timeout)
```

With this suggestion, there is no need to change the `config` package.

Comment (RamLavi) [reply pkg/mainflow.go]: DONE

**Comment by orelmisan** [comment]
There are two commits, the first looks like a leftover.

File: pkg/mainflow.go [hunk 2]
```diff
@@ -64,8 +64,9 @@ func Run(rawEnv map[string]string, namespace string) error {
 	return l.Run(ctx)
 }
 
-func printConfig(checkupConfig config.Config) {
+func printConfig(baseConfig kconfig.Config, checkupConfig config.Config) {
 	log.Println("Using the following config:")
+	log.Printf("%q: %q", baseConfig.Timeout, baseConfig.Timeout)
```
Comment (orelmisan) [inline pkg/mainflow.go:69]: I think this line will print the time out value twice.

```suggestion
	log.Printf("%q: %q", "timeout", baseConfig.Timeout)
```

**Review by orelmisan — COMMENTED** [review]
Thank you for the changes.
Could you please update the PR description after the new changes?

**Comment by RamLavi** [comment]
Passed on CNV 4.14 (nightly 4.14.0-0.nightly-2023-07-20-215234) with [OCPBUGS-15102](https://issues.redhat.com/browse/OCPBUGS-15102)'s workaround:
```
docker run --rm \
           --volume /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:Z \
           --volume /home/ralavi/.kube:/root/.kube:Z,ro \
           --workdir /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup \
           -e KUBECONFIG=/root/.kube/kubeconfig \
           -e TEST_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup:latest \
           -e TEST_NAMESPACE=dpdk-checkup-ns-1 \
           -e NETWORK_ATTACHMENT_DEFINITION_NAME=dpdk-sriovnetwork-ns-1 \
           -e TRAFFIC_GEN_IMAGE_URL=quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main \
           -e VM_CONTAINER_DISK_IMAGE_URL=quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main \
           docker.io/library/golang:1.20.6 go test ./tests/... -test.v -test.timeout=1h -ginkgo.v -ginkgo.timeout=1h
=== RUN   TestKubevirtDpdkCheckup
Running Suite: KubevirtDpdkCheckup Suite - /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests
==============================================================================================================
Random Seed: 1690957294

Will run 1 of 1 specs
------------------------------
[BeforeSuite] 
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/test_suite_test.go:60
[BeforeSuite] PASSED [0.002 seconds]
------------------------------
Execute the checkup Job should complete successfully
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/checkup_test.go:79
• [248.126 seconds]
------------------------------

Ran 1 of 1 Specs in 248.129 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestKubevirtDpdkCheckup (248.13s)
PASS
ok  	github.com/kiagnose/kubevirt-dpdk-checkup/tests	248.143s
```

File: pkg/mainflow.go (hunk 2 above)
Comment (RamLavi) [reply pkg/mainflow.go]: good catch!

**Comment by RamLavi** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/20a3474f97db25d107f5da45d309a2df8065a1e8..11f991cb9732ac80b6b7324a88f9f839c2cfce42): Fix review

**Comment by RamLavi** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/11f991cb9732ac80b6b7324a88f9f839c2cfce42..5852eebdab1eaff77787ce67b3b8eb40acad7a9f): Rebase

**Review by orelmisan — APPROVED** [review]
Thank you

**Pull request merged**